ethdb/privateapi/ethbackend.go

```diff
@@ -459,3 +481,22 @@ func (s *EthBackendServer) NodeInfo(_ context.Context, r *remote.NodesInfoReques
 	}
 	return nodesInfo, nil
 }
+
+func (s *EthBackendServer) unwindCycle(unwind_point uint64) {
+	// Discard all payload assembled since new state has been generated
+	s.pendingPayloads = make(map[uint64]types2.ExecutionPayload)
+	s.unwindForkChoicePOSCh <- unwind_point
```

Comment:
Concurrency logic here doesn't make sense to me:

1. If you use a channel (`unwindForkChoicePOSCh`) to send a request, then be consistent and use a channel to get a response instead of using `sync.Cond`. Using channels is more idiomatic in go, especially given that we already use channels to communicate between `EthBackendServer` and synced stage.
2. Creating and using a channel only locally (`unwindFinished`) doesn't make sense.
3. Using a timer is a bad way to patch concurrency issues.